Tasks/MobileCenterDistribute/mobilecenterdistribute.ts

```diff
@@ -352,7 +352,7 @@ async function run() {
         if (!userAgent) {
             userAgent = 'VSTS';
         }
-        userAgent = userAgent + ' (Task:VSMobileCenterUpload)';
+        userAgent = userAgent + ' (Task:MobileCenterDistribute)';
```

Comment:
This user agent string change will impact telemetry that Mobile Center team looks at. Let us leave it as is till we confirm with them.